### ISSUE TITLE
[a11y] Filters on CRT view missing labels 

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#288]
When the field (or selection) is highlighted, it's not reading the label for screen reader users. 
- [x] Filter by section ([here's an example I found before](https://a11y-guidelines.orange.com/web_EN/exemples/tag-exemples/Exemple_filtre.html))
- [x] Contact first name
- [x] Contact last name
- [x] Incident city - marked as required, not sure why
- [x] Incident state - marked as required, but not required
- [x] remove filter tag reads data item instead of plain language text. ex: reads "remove filter **assigned_section**: ELS", expected "remove filter **assigned section**: ELS" -- it's reading the underscore

**Comment by Jacklynn** [comment]
@el-mapache I reviewed these and added comments above!

**Comment by Jacklynn** [comment]
I just checked these and the problems are still there

**Comment by blaughman** [comment]
Filters have labels.  Moving to UAT.

**Comment by Jacklynn** [comment]
This is for screen readers, but I just checked with VO and it seems to all be working!
